java/code/webapp/WEB-INF/nav/system_detail.xml

```diff
@@ -167,7 +167,7 @@
     </rhn-tab>
   </rhn-tab>
 
-  <rhn-tab name="Groups" acl="system_feature(ftr_system_grouping); user_role(org_admin) or user_role(system_group_admin);system_has_management_entitlement() or system_has_salt_entitlement() or system_is_bootstrap_minion_server()" url="/rhn/systems/details/groups/ListRemove.do">
+  <rhn-tab name="Groups" acl="system_feature(ftr_system_grouping); user_role(org_admin) or user_role(system_group_admin);system_has_management_entitlement() or system_has_salt_entitlement() or system_is_bootstrap_minion_server() or system_has_foreign_entitlement()" url="/rhn/systems/details/groups/ListRemove.do">
```

Comment:
I wonder if this check is really needed anymore. There is no other base entitlement left.

Reply:
This will also affect systems added by the virtual host gatherer: they are added as foreign. Is it OK to be able to add those VMware machines in a group?

Reply:
The groups are just shown in UI and API. Additional functionality like group pillars is enabled only with the minion entitlement.
So I can't see anything that could break.